torchmetrics/utilities/data.py

```diff
@@ -151,20 +152,29 @@ def get_num_classes(
     return num_classes
 
 
-def _stable_1d_sort(x: torch, N: int = 2049):
+def _stable_1d_sort(x: torch, nb: int = 2049):
```

Comment:
Have you checked if this is used with keywords somewhere?

Reply:
yes, nowhere used, that is why we did not the bug :D

Reply:
moreover, curious why it is needed since it gives the same results with and without passing any number...